client/client.go

```diff
@@ -452,3 +406,17 @@ func GetB64MD5Hash(name string) (b64Hash string, rerr error) {
 
 	return base64.StdEncoding.EncodeToString(h.Sum(nil)), err
 }
+
+// GetUploadURLRegion returns the correct base url depending on the env variable CLOUDABILITY_UPLOAD_REGION.
+// If value is not supported, default to us-west-2 (original) URL
+func GetUploadURLRegion(region string) string {
```

Comment:
Nit, s/GetUploadURLRegion/GetUploadURL. The previous name sounds like return region value